openid-sse-framework-1_0.txt

```diff
@@ -1504,21 +1617,182 @@ Tulshibagwale, et al.   Expires December 10, 2021              [Page 26]
 
 
 
-Tulshibagwale, et al.   Expires December 10, 2021              [Page 27]
+
+
+
+
+Tulshibagwale, et al.        Standards Track                   [Page 29]
 
                           openid-sse-framework                 June 2021
 
 
-   +------+------------------------------------------------------------+
-   | Code | Description                                                |
-   +------+------------------------------------------------------------+
-   | 401  | if authorization failed or it is missing                   |
-   |      |                                                            |
-   | 403  | if the Event Receiver is not allowed to update the stream  |
-   |      | configuration                                              |
-   +------+------------------------------------------------------------+
+     +======+========================================================+
+     | Code | Description                                            |
+     +======+========================================================+
+     | 401  | if authorization failed or it is missing               |
+     +------+--------------------------------------------------------+
+     | 403  | if the Event Receiver is not allowed to read the       |
+     |      | stream status                                          |
+     +------+--------------------------------------------------------+
+     | 404  | if there is no Event Stream with the given "stream_id" |
+     |      | for this Event Receiver, or if the Subject specified   |
+     |      | is invalid or if the Receiver is not authorized to get |
+     |      | status for the specified Subject.                      |
+     +------+--------------------------------------------------------+
+
+                     Table 6: Read Stream Status Errors
+
+   Examples:
+
+   1.  If a Receiver makes a request with an invalid OAuth token, then
+       the Transmitter MUST respond with a 401 error status.
+
+   2.  If the Receiver presents a valid OAuth token, but the Transmitter
+       policy does not permit the Receiver from obtaining the status,
+       then the Transmitter MAY respond with a 403 error status.
+
+   3.  If the Receiver requests the status for a stream that does not
+       exist then the Transmitter MUST respond with a 404 error status.
+
+   4.  If the Receiver requests the status for specific Subject, but the
+       Transmitter policy does not permit the Receiver to read the
+       status of that Subject, then the Transmitter MAY respond with a
+       404 error status in order to not reveal the policy decision.
+
+   5.  If the specified Subject is invalid then the Transmitter MUST
+       respond with a 404 error status.
+
+7.1.2.2.  Updating a Stream's Status
+
+   An Event Receiver updates the current status of a stream by making an
```

Comment:
we should mark this for a later pull request, but this needs to be changed to a PATCH since it's an update.

Reply:
👍 I'll make an issue for this and the examples comment above.